roles/pulp/defaults/main.yaml

```
@@ -14,5 +14,7 @@ pulp_content_container_name: pulp-content
 pulp_worker_container_name: pulp-worker
 
 pulp_content_origin: "http://{{ ansible_fqdn }}:24816"
+pulp_rhsm_url: "https://{{ ansible_fqdn }}/rhsm"
+pulp_pulp_url: "https://{{ ansible_fqdn }}"
```

Keeping in line with the origin that's on port 24816, should this match the default of the all-in-one container?
```suggestion
pulp_pulp_url: "http://{{ ansible_fqdn }}:24817"
```

---

Yeah